chunk/inmemory_storage_client.go

```diff
@@ -125,7 +125,8 @@ func (m *MockStorage) BatchWrite(_ context.Context, batch WriteBatch) error {
 			items = append(items, mockItem{})
 			copy(items[i+1:], items[i:])
 		} else {
-			return fmt.Errorf("Dupe write")
+			// Ignore duplicate write
+			continue
```

Comment:
Why are we doing duplicate writes?  This will break DynamoDB if we do a dupe write in the same batch.  Need to put the error back in, and add this as a comment.